Libraries/Components/View/View.js

```diff
@@ -161,6 +164,7 @@ const View: React.AbstractComponent<
           accessibilityElementsHidden={
             ariaHidden ?? accessibilityElementsHidden
           }
+          accessibilityLabelledBy={_accessibilityLabelledBy}
```

Comment:
L174 is overriding this and some of the other props. The restProps stuff is a bit of a mess right now. Please could you fix that in this PR by moving all the prop extraction (including for accessibility value, state, etc) up to the block that is around L47. Then "otherProps" should be spread into the native component before all the specific accessibility props too. This is the kind of bug that snapshot tests could be detecting